### PR TITLE
Adding option for triggering on Lc->pKpi and Lc->pK0s

### PR DESCRIPTION
**Pull request by arossi81**

### CODE REVIEW

**Comment by arossi81** [comment]
wait before approving it, I want to make a change

**Comment by arossi81** [comment]
Now it is ok.

File: MC/CustomGenerators/PWGHF/Pythia8_Monash2013_D2Hdedicated.C [hunk 1]
```diff
@@ -78,8 +86,8 @@ AliGenerator *GeneratorCustom(TString opt = "")
     (AliPythia8::Instance())->ReadString("333:onMode = off");
     (AliPythia8::Instance())->ReadString("333:onIfAll = 321 321");
     //add Xic+ decays absent in PYTHIA8 decay table
-    (AliPythia8::Instance())->ReadString("4232:addChannel = 1 0.2 0 2212 313");
-    (AliPythia8::Instance())->ReadString("4232:addChannel = 1 0.2 0 2212 321 211");
+    (AliPythia8::Instance())->ReadString("4232:addChannel = 1 0.2 0 2212 -313");
```
Comment (chiarazampolli) [inline MC/CustomGenerators/PWGHF/Pythia8_Monash2013_D2Hdedicated.C:89]: you change also the Xic part, right?

**Comment by chiarazampolli** [comment]
Looks fine to me. @gconesab , @fprino ?

Chiara

**Comment by fprino** [comment]
Hi,

for the Xic0 decay, we need to modify the code in AliDecayerPythia8, not only in the Config macro. In fact, for most of the AliRoot versions that we us in production, the check
if(AliPythiaBase::Class()->GetMethodAny("Decayer")){
will return kTRUE and will use the ForceHadronicD of AliDecayerPythia8, where the signs are also wrong.

Ciao, Francesco

**Comment by chiarazampolli** [comment]
Ciao Francesco,

Here I was mainly referring to the fact that Andrea said in JIRA that he used ForceHadronicD in this PR, but this is not the case. I did not consider the sign issue for the time being. I want to be sure that this PR is what Andrea meant.

Chiara

**Comment by arossi81** [comment]
Ciao @chiarazampolli ,@fprino,
this commit is meant just for triggering on Lc. It also has the proper signs on the Xic pKpi decay channels but, as Francesco said, this won't be effective, I changed this part just for completeness. We need to change AliDecayerPythia8 to fix that issue. however, this is not a problem/stopper for the ITS3 production (we trigger on Lc), but for the Xic production.

Ciao
Andrea

**Comment by arossi81** [comment]
> Ciao Francesco,
> 
> Here I was mainly referring to the fact that Andrea said in JIRA that he used ForceHadronicD in this PR, but this is not the case. I did not consider the sign issue for the time being. I want to be sure that this PR is what Andrea meant.
> 
> Chiara

Ciao Chiara, in the Config I did use the ForceHadronicD for the Lc, since
 if(AliPythiaBase::Class()->GetMethodAny("Decayer"))
will pass the selection for all recent AliRoot version

Cheers
Andrea

**Comment by chiarazampolli** [comment]
Ciao @arossi81 , 

Ok, so the fact that there is:

`pyth->SetForceDecay(...)`

does not matter, right? Sorry, I never configured Pythia8.

Cheers,

Chiara

**Comment by arossi81** [comment]
ciao @chiarazampolli ,
yes, it matters because that is the option that will be used . For consistency with the other channels I included the definition of the decay channels also in the Config, but what will be used is the ForceHadronicD method, activated via the call to pyth->SetForceDecay(kLcpKpi).
Ciao
Andrea

**Comment by chiarazampolli** [comment]
Ok, so SetForceDecay() will call SetForceDecayD(). 

For me we can merge. @fprino , more comments? This config is not for the Xic, and anyway, the fix for that is in AliRoot.

Chiara

**Comment by fprino** [comment]
Hi,

yes, I agree. I am going to merge this PR now.

Ciao, Francesco

**Pull request merged**